cmd/gpmt/root.go

```diff
@@ -0,0 +1,110 @@
+/*
+Greenplum Magic Tool
+
+Authored by Tyler Ramer, Ignacio Elizaga
+Copyright 2018
+
+Licensed under the Apache License, Version 2.0 (the "License")
+
+*/
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/pivotal-gss/gpmt2/pkg/db"
+	"github.com/rifflock/lfshook"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// Tool information constants
+
+const (
+	gpmtVersion = "Version (pre)ALPHA"
+	githubRepo  = "https://github.com/pivotal-gss/gpmt2"
+)
+
+type logOptions struct {
+	Verbose bool
+	LogDir  string
+	LogFile string
+}
+
+// Local Package Variables
+var (
+	// gp_log_collector flags
+	lcOpts LogCollectorOptions
+
+	// DB connection details
+	connString db.ConnString //FIXME/TODO: Do we need a separate wrapper for DB?
```

Comment:
Did you mean wrapper in the sense ( db.go ) or the go type struct ?

Reply:
I mean in the sense of a db.go file. Maybe we would move it to the internal folder, if we need some easy way of executing commands. But the database/sql golang package is already fairly obfuscated. We will consider further in time.